KBoard/KBoard/Controller/MyBoard/CategoryViewController.swift

```diff
@@ -47,77 +53,114 @@ class CategoryViewController: UIViewController {
     }
 
     @objc fileprivate func editWords() {
-        // TODO: Edit Words
-//        let categoryEditViewController = CategoryEditViewController()
-//        navigationController?.pushViewController(categoryEditViewController, animated: true)
+        tableView.setEditing(!tableView.isEditing, animated: true)
+        if tableView.isEditing {
+            navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .done, target: self, action: #selector(editWords))
+        } else {
+            navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .edit, target: self, action: #selector(editWords))
+        }
```

Comment:
```suggestion
navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: tableView.isEditing ? .done : .edit, target: self, action: #selector(editWords))
```

분기 처리라면 삼항연산자로 라인 수를 줄여보는건 어떤가요?!👍